packages/firestore/test/integration/api/query.test.ts

```diff
@@ -2203,6 +2203,153 @@ apiDescribe('Queries', persistence => {
     }
   ).timeout('90s');
 
+  // TODO(b/291365820): Stop skipping this test when running against the
+  // Firestore emulator once the emulator is improved to include a bloom filter
+  // in the existence filter messages that it sends.
+  // eslint-disable-next-line no-restricted-properties
+  (USE_EMULATOR ? it.skip : it)(
+    'bloom filter can correctly handle updated and newly added documents',
```

Comment:
Consider renaming this test to something like this:

> bloom filter should avert a full requery when documents were added, deleted, removed, updated, and unchanged since the resume token